### DISCUSSION TITLE
Are there any plans to enable lora for lama.cpp?

### DISCUSSION THREAD

**Discussion by Taikono-Himazin** [oobabooga/text-generation-webui#4818] [Q&A]
Looking at this table, lora is not supported when the Model loader is lama.cpp.
https://github.com/oobabooga/text-generation-webui/wiki

It is already possible to load lora in lama.cpp.
https://github.com/ggerganov/llama.cpp/tree/master/examples/finetune
I think it's just a matter of adding it to the argument, but are there any plans for this improvement?

Also, lama.cpp can load multiple loras and change the strength with which each is applied.

**Comment by Gee1111** [comment]
+1
